tr/django_start_project/README.md

```diff
@@ -14,16 +14,16 @@ Bazı dosya ve dizinlerin isimleri Django için çok önemlidir. Oluşturmak üz
 
 > virtualenv içindeki her şeyi çalıştırmayı unutmayın. Eğer konsolunuzda bir önek `(myvenv)` görmüyorsanız virtualenv'nizi çalışır hale getirmelisiniz. **Django yükleme** bölümünün **virtualenv ile Çalışma** kısmında nasıl yapılacağını açıkladık. Windows'da ` myvenv\Scripts\activate` ya da Mac OS / Linux'ta `source myvenv/bin/activate` yazmak sizin için bunu yapacaktır.
 
-MacOS veya Linux konsolunuzda aşağıdaki komutu çalıştırmalısınız; **sonuna nokta `(.)` koymayı unutmayın**:
+MacOS veya Linux konsolunuzda aşağıdaki komutu çalıştırmalısınız; **sonuna nokta `( .)` koymayı unutmayın**:
```

Comment:
I am almost sure you don't need the whitespace before the dot here and in on further places. Why do you think it needs being added?